docs/nlu/choosing-a-pipeline.rst

```diff
@@ -36,19 +42,33 @@ use the ``supervised_embeddings`` pipeline:
 A Longer Answer
 ---------------
 
-The two most important pipelines are ``supervised_embeddings`` and ``pretrained_embeddings_spacy``.
-The biggest difference between them is that the ``pretrained_embeddings_spacy`` pipeline uses pre-trained
-word vectors from either GloVe or fastText. The ``supervised_embeddings`` pipeline, on the other hand,
-doesn't use any pre-trained word vectors, but instead fits these specifically for your dataset.
+The three most important pipelines are ``supervised_embeddings``, ``pretrained_embeddings_convert`` and ``pretrained_embeddings_spacy``.
+The ``pretrained_embeddings_spacy`` pipeline uses pre-trained
+word vectors from either GloVe or fastText, whereas ``pretrained_embeddings_convert`` uses a pretrained sentence encoding model `ConveRT <https://github.com/PolyAI-LDN/polyai-models>`_ to
+extract vector representations of complete user utterance as a whole. On the other hand, the ``supervised_embeddings`` pipeline
+doesn't use any pre-trained word vectors or sentence vectors, but instead fits these specifically for your dataset.
 
 
 pretrained_embeddings_spacy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The advantage of the ``pretrained_embeddings_spacy`` pipeline is that if you have a training example like:
+The advantage of ``pretrained_embeddings_spacy`` pipeline is that if you have a training example like:
 "I want to buy apples", and Rasa is asked to predict the intent for "get pears", your model
 already knows that the words "apples" and "pears" are very similar. This is especially useful
-if you don't have very much training data.
+if you don't have large enough training data.
+
+
+pretrained_embeddings_convert
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The advantage of the ``pretrained_embeddings_convert`` pipeline is that it doesn't treat each word of the user message independently,
+but creates a contextual vector representation for the complete sentence. For example, if you have a training example, like:
+"can I book a car?", and Rasa is asked to predict the intent for "I need a ride from my place", since the contextual vector representation for both
```

Comment:
did you check this example?

Reply:
Yes. ConveRT uses this example in their example notebook to demo sentence  similarity.